docs/using_packet.md

```diff
@@ -29,7 +29,7 @@ version = "0.1.0"
 authors = ["My Name <my.email@mydomain.com>"]
 build = "build.rs"
 [build-dependencies]
-syntex = "0.31.*"
+syntex = "0.42.*" # syntex version must match the version being used in pnet_macros/Cargo.toml
```

Comment:
Specifying a number here at all is just doomed to become outdated sooner or later. IMO it's better to say something like
`syntex = "X" # Where X must be the syntex version being used in pnet_macros/Cargo.toml`